apps/web/src/pages/Pools/PoolPage/PoolPage.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { useParams, Outlet } from "react-router-dom";
+import {
+  ClipboardDocumentIcon,
+  Cog8ToothIcon,
+  UserGroupIcon,
+} from "@heroicons/react/24/outline";
+
+import PageHeader from "@common/components/PageHeader";
+import Pending from "@common/components/Pending";
+import SEO from "@common/components/SEO/SEO";
+import { ThrowNotFound } from "@common/components/NotFound";
+import { getFullPoolAdvertisementTitleLabel } from "@common/helpers/poolUtils";
+
+import useRoutes from "~/hooks/useRoutes";
+import useCurrentPage from "~/hooks/useCurrentPage";
+import usePageSubNav from "~/hooks/usePageSubNav";
+import { Pool, useGetBasicPoolInfoQuery } from "~/api/generated";
+import { PageNavInfo } from "~/types/pages";
+
+type PageNaveKeys = "view" | "edit" | "candidates";
+
+interface PageContentProps {
+  pool: Pick<Pool, "id" | "classifications" | "stream" | "name">;
+}
+
+const PageContent = ({ pool }: PageContentProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const pages = new Map<PageNaveKeys, PageNavInfo>([
+    [
+      "view",
+      {
+        icon: ClipboardDocumentIcon,
+        title: intl.formatMessage({
+          defaultMessage: "Pool information",
+          id: "Cjp2F6",
+          description: "Title for the pool info page",
+        }),
+        link: {
+          url: paths.poolView(pool.id),
+        },
+      },
+    ],
+    [
+      "edit",
+      {
+        icon: Cog8ToothIcon,
+        title: intl.formatMessage({
+          defaultMessage: "Edit pool",
+          id: "l7Wu86",
+          description: "Title for the edit pool page",
+        }),
+        link: {
+          url: paths.poolUpdate(pool.id),
+        },
+      },
+    ],
+    [
+      "candidates",
+      {
+        icon: UserGroupIcon,
+        title: intl.formatMessage({
+          defaultMessage: "Candidates",
+          id: "X4TOhW",
+          description: "Page title for the admin pool candidates index page",
+        }),
+        link: {
+          url: paths.poolCandidateTable(pool.id),
+          label: intl.formatMessage({
+            defaultMessage: "View Candidates",
+            id: "Rl+0Er",
+            description: "Title for the edit pool page",
+          }),
+        },
+      },
+    ],
+  ]);
+
+  const poolTitle = getFullPoolAdvertisementTitleLabel(intl, pool);
+  const currentPage = useCurrentPage<PageNaveKeys>(pages);
+  const navItems = usePageSubNav<PageNaveKeys>(pages);
+
+  return (
+    <>
+      <SEO title={currentPage?.title} />
+      <PageHeader
+        subtitle={poolTitle}
+        icon={currentPage?.icon}
+        navItems={navItems}
+      >
+        {currentPage?.title}
+      </PageHeader>
+    </>
+  );
+};
+
+const PoolPage = () => {
```

Comment:
Okay, I think I understand what's going on now, but I have to admit I got pretty confused looking at this at first 😂. I was thrown off by the "PageContent" component containing the PageHeader but no actual content, and unsure about this Page's relationship to ViewPoolPage and EditPoolPage.

To make the relationship more clear, I would suggest renaming this something like PoolPageWrapper and bumping this file up a directory. It's not really a sibling to the other Pages, or a full page itself, its more of a wrapper after all.

Reply:
Yeahh, I was really struggling with this one. I think this is a good suggestion 👍

Reply:
Okay, so I have been thinking about this one. I agree about bumping it up as for naming what about:

- `PoolHeader`
- `PoolLayout`

We are using the idea of `<Layout />` already to wrap the entire app so it may make sense here to and since the "content" really is just the page header `<PoolHeader />` makes sense since it is just a more specific `<PageHeader />`

Reply:
Sounds good to me